.github/actions/validate-with-schema/README.md

```diff
@@ -0,0 +1,32 @@
+# Validate With Schema Action
+
+This action validates a `json` or `yaml` file using a given schema. Essentially acts as a convenience wrapper over `GrantBirki/json-yaml-validate`.
+
+## Inputs
+
+| Name | Type | Description | Required | Default | Example |
+| ---- | ---- | ----------- | -------- | ------- | ------- |
+| `schema-version` | `string` | Version of the schema required in SchemaVer | `true` | N/A | `"1-0-0"` |
+| `schema-location` | `string` | Directory within access-nri/schema that contains the schema | `true` | N/A | `"access-nri.org.au/deployment/config/versions"` |
+| `data-location` | `string` | Path to the json/yaml data in the callers repository | `true` | N/A | `"./some/data.json"` |
```

Comment:
Maybe put a note that it relies on YAML files have a subscript of `.yml` or `.yaml`?

Reply:
Thanks for that, added!